common/SmokeTests/SmokeTest/SmokeTest.csproj

```diff
@@ -14,11 +14,11 @@
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.4.0-dev.*" />
     <PackageReference Include="Azure.Identity" Version="1.2.0-dev.*" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.1.0-dev.*" />
-    <PackageReference Include="Azure.Messaging.EventHubs.Processor" Version="5.1.0-dev.*" />
+    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.3.0-dev.*" />
```

Comment:
I thought this got overwritten by a script to pull the latest from the dev feed for each run....

Reply:
@benbp....  am I misremembering here?

Reply:
```
Setting Azure.Core to 1.3.0-dev.20200624.1
Using baseline version date: 20200624
Setting Azure.Identity to 1.2.0-dev.20200624.1
Setting Azure.Messaging.EventHubs to 5.2.0-dev.20200624.1
Setting Azure.Messaging.EventHubs.Processor to 5.2.0-dev.20200624.1
Setting Azure.Security.Keyvault.Secrets to 4.1.0-dev.20200624.1
Setting Azure.Storage.Blobs to 12.5.0-dev.20200624.1

```
The script is broken. I'll take a look.

Reply:
@jsquire it does, but we were also updating what was on disk just for local dev scenarios. I've been waiting until after the current release cycle (yesterday/today?) to merge https://github.com/Azure/azure-sdk-for-net/pull/17524. @pakrym I've got a lot of updates to the update-dependencies script in that PR, but I can rebase them against your changes if you merge first.

Reply:
@pakrym the script has a bug to pull old dependency versions for the daily tests. My PR I linked in the comment above fixes this issue.

Reply:
I'll try to merge this one if you don't mind.

Reply:
Or not, I think I'm digging myself deeper and deeper into the issues you've already solved.